src/assets/i18n/fr.json5

```diff
@@ -3251,48 +3712,49 @@
   "mydspace.status.workflow": "Workflow",
 
   // "mydspace.status.workspace": "Workspace",
-  "mydspace.status.workspace": "Workspace",
+  "mydspace.status.workspace": "Espace de travail",
```

Comment:
This tag is used to mention that a submission is started but as not been deposited yet. Maybe "Dépôts non complété".

Reply:
@pilasou In our Dspace 5.9 instance we use the title "Soumissions interrompues/retournées" for this section. I think I would stick to "Espace de travail", so as not to stray too far from the original English term. May I suggest this kind of modification be made in you local version maybe?

Reply:
@pilasou and @mhvezina : If I may chime in here briefly (though not on the translation itself, as I'm not any help there).  _I'd highly recommend not striving for perfection in any single translation._  Most sites *will modify* the translation if it doesn't suite them exactly.  We just want a French translation that should make sense to most French users, even if it may not be perfect for specific dialects of French.  It's the same with the original English.  It's understandable to people who speak/read English, but different English dialects (e.g. UK vs USA) may wish to modify the English text for local purposes.

I should also add, it's also possible to have two versions of French if we ever find it necessary to do so.  That's already done for Portuguese, where we have a Brazilian version (`pt-BR.json5`) and a Portugal version (`pt-PT.json5`), simply because the difference in dialects was important (and we had people interested in creating a dialect specific version).

Reply:
@tdonohue yes, you are right (about the possible local version and about not trying to have a near perfect translation from the first attempt). I talked to @pilasou off the list and we agreed. Yes, maybe we could think of a fr-CA translation, although there wouldn't not be that many differences. I guess a unique French translation is ok for now.